Lib9c/Action/IJoinArena.cs

```diff
@@ -0,0 +1,19 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using Libplanet;
+using BencodexList = Bencodex.Types.List;
+
+namespace Nekoyume.Action
+{
+    public interface IJoinArena
+    {
+        Address AvatarAddress { get; }
+        int ChampionshipId { get; }
+        int Round { get; }
+        IEnumerable<Guid> Costumes { get; }
+        IEnumerable<Guid> Equipments { get; }
+        IEnumerable<BencodexList>? RuneSlotInfos => null;
```

Comment:
It's ok to assume type as `BencodexList` but can we guarantee given `RuneSlotInfos` is valid? or is up to user?

Reply:
It's up to the users. It will guarantee the `RuneSlotInfos` but is not type level. Also, it can be broken when the rune slot info model changes.